src/app/profile/payment-methods/payment-methods.component.js

```diff
@@ -54,14 +54,14 @@ class PaymentMethodsController {
     this.$rootScope.$on( SignOutEvent, ( event ) => this.signedOut( event ) );
 
     this.loading = true;
-    this.loadPaymentMethods();
-    this.loadDonorDetails();
```

Comment:
Yes, the SessionEnforcer by default calls the sign-in callback on init if the session is signed in. It's safe to remove the calls from $onInit.